spec/observables/dom/fetch-spec.ts

```diff
@@ -173,13 +177,13 @@ describe('fromFetch', () => {
   });
 
   it('should allow passing of init object', done => {
-    const myInit = {};
-    const fetch$ = fromFetch('/foo', myInit);
+    const myInit = {myInit: true};
+    const fetch$ = fromFetch('/foo', myInit as {});
     fetch$.subscribe({
       error: done,
       complete: done,
     });
-    expect(mockFetch.calls[0].init).to.equal(myInit);
+    expect((mockFetch.calls[0].init as any).myInit).to.be.true;
     expect(mockFetch.calls[0].init.signal).not.to.be.undefined;
   });
```

Comment:
I would like to see the test description changed to specifically state that the `init` object should not be mutated.

Also, I'd like to see `expect(mockFetch.calls[0].init).to.not.equal(myInit);` added as an expectation.

And I would like to see one of the actual `RequestInit` properties used instead of the invalid `myInit` property. For example: `const myInit = { method: 'HEAD' }`

Reply:
Thanks for the feedback! I've updated the tests